SU-SWS/react_paragraphs
# D8CORE-2106

# READY FOR REVIEW

# Summary
- The link field widget needs to be able to support cardinality in order to allow more than one link to be used in the CTA List paragraph type.

# Review By (Date)
- ASAP

# Urgency
- These changes unblocks D8CORE-1656

# Steps to Test

1. Build a stack with this branch of the react_paragraphs module.
2. Add a paragraph with a link field (or create a new paragraph type with a link field)
3. Add a link.
4. Click the "Add Another Link" button
5. Notice it adds an additional link to the list.  
6. Add an additional link.  Save the paragraph/page.
7. Edit the paragraph.  Click on the "Remove" button next to the second link's `Link Text` field.
8. Notice it removes the additional link.
9. Save the paragraph.  Make sure it displays only your first link correctly. 

# Affected Projects or Products
- SOE custom paragraph types.

# Associated Issues and/or People
- D8CORE-1656

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
